### PR TITLE
Add metrics for azure disk attach and detach operations

### PR DESCRIPTION
**Pull request by nilo19**
**What type of PR is this?**

/kind feature

**What this PR does / why we need it**:

Add metrics for azure disk attach and detach operations.

**Which issue(s) this PR fixes**:

Fixes #

**Requirements**:
- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Special notes for your reviewer**:

**Release note**:
```
Add metrics for azure disk attach and detach operations.
```

### CODE REVIEW

File: pkg/azuredisk/controllerserver.go [hunk 1]
```diff
@@ -429,11 +438,18 @@ func (d *Driver) ControllerPublishVolume(ctx context.Context, req *csi.Controlle
 	}
 
 	pvInfo := map[string]string{LUN: strconv.Itoa(int(lun))}
+	isOperationSucceeded = true
 	return &csi.ControllerPublishVolumeResponse{PublishContext: pvInfo}, nil
 }
 
 // ControllerUnpublishVolume detach an azure disk from a required node
 func (d *Driver) ControllerUnpublishVolume(ctx context.Context, req *csi.ControllerUnpublishVolumeRequest) (*csi.ControllerUnpublishVolumeResponse, error) {
+	mc := metrics.NewMetricContext("azuredisk_csi_driver", "controller_unpublish_volume", d.cloud.ResourceGroup, d.cloud.SubscriptionID, d.Name)
```
Comment (andyzhangx) [inline pkg/azuredisk/controllerserver.go:447]: use const `DriverName` intead of "azuredisk_csi_driver"

**Comment by andyzhangx** [comment]
our e2e test logs already has the metrics:
```
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="0.1"} 1
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="0.2"} 2
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="0.5"} 2
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="1"} 2
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="10"} 2
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="20"} 12
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="30"} 13
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="40"} 13
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="50"} 25
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="60"} 26
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="100"} 29
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="200"} 30
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="300"} 30
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="+Inf"} 30
cloudprovider_azure_op_duration_seconds_sum{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e"} 1030.1893204469998
cloudprovider_azure_op_duration_seconds_count{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e"} 30
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="0.1"} 0
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="0.2"} 0
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="0.5"} 0
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="1"} 0
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="10"} 0
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="20"} 17
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="30"} 20
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="40"} 24
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="50"} 27
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="60"} 27
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="100"} 27
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="200"} 27
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="300"} 27
cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="+Inf"} 27
cloudprovider_azure_op_duration_seconds_sum{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e"} 560.8781047589999
cloudprovider_azure_op_duration_seconds_count{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e"} 27
```

File: pkg/azuredisk/controllerserver.go (hunk 1 above)
Comment (andyzhangx) [reply pkg/azuredisk/controllerserver.go:447]: `DriverName` is `disk.csi.azure.com`, use a const var(`azuredisk_csi_driver`) is more appropiate

**Comment by nilo19** [comment]
> our e2e test logs already has the metrics:
> 
> ```
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="0.1"} 1
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="0.2"} 2
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="0.5"} 2
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="1"} 2
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="10"} 2
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="20"} 12
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="30"} 13
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="40"} 13
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="50"} 25
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="60"} 26
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="100"} 29
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="200"} 30
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="300"} 30
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="+Inf"} 30
> cloudprovider_azure_op_duration_seconds_sum{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e"} 1030.1893204469998
> cloudprovider_azure_op_duration_seconds_count{request="azuredisk_csi_driver_controller_publish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e"} 30
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="0.1"} 0
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="0.2"} 0
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="0.5"} 0
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="1"} 0
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="10"} 0
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="20"} 17
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="30"} 20
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="40"} 24
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="50"} 27
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="60"} 27
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="100"} 27
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="200"} 27
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="300"} 27
> cloudprovider_azure_op_duration_seconds_bucket{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e",le="+Inf"} 27
> cloudprovider_azure_op_duration_seconds_sum{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e"} 560.8781047589999
> cloudprovider_azure_op_duration_seconds_count{request="azuredisk_csi_driver_controller_unpublish_volume",resource_group="kubetest-717833f9-f276-11ea-9029-32d2d2a853c2",source="disk.csi.azure.com",subscription_id="0e46bd28-a80f-4d3a-8200-d9eb8d80cb2e"} 27
> ```

yeah that's what i'm doing

**Comment by andyzhangx** [comment]
windows test is failing again cc @ZeroMagic 
```
  /home/prow/go/src/sigs.k8s.io/azuredisk-csi-driver/test/e2e/dynamic_provisioning_test.go:41
    should create a statefulset object, write and read to it, delete the pod and write and read to it again [kubernetes.io/azure-disk] [disk.csi.azure.com] [Windows] [It]
    /home/prow/go/src/sigs.k8s.io/azuredisk-csi-driver/test/e2e/dynamic_provisioning_test.go:574

    Unexpected error:
        <*errors.errorString | 0xc0007c6dd0>: {
            s: "Failed to find \"hello world\r\nhello world\r\n\", last result: \"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00hello world\r\n\"",
        }
        Failed to find "hello world
        hello world
        ", last result: "�������������hello world
        "
    occurred

    /home/prow/go/src/sigs.k8s.io/azuredisk-csi-driver/test/e2e/testsuites/testsuites.go:632
```

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Pull request merged**